### PR TITLE
Version 17.12

### PR DESCRIPTION
**Pull request by AR-May**

### CODE REVIEW

**Review by rainersigwald — COMMENTED** [review]
LGTM but before we can merge we need to make sure the darc flows are set up for 17.11 and 17.12, and the VS insertion flow is ready to handle 17.12 bits coming from our main.

**Comment by AR-May** [comment]
The darc flows for 17.11 and 17.12 are fully set up and the VS insertion flows are nonexistent - I disabled both and from now will make the releases manually until VS is ready to receive 17.12

**Review by rainersigwald — APPROVED** [review]

File: eng/Versions.props [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.11.0</VersionPrefix>
-    <PackageValidationBaselineVersion>17.8.3</PackageValidationBaselineVersion>
+    <VersionPrefix>17.12.0</VersionPrefix>
+    <PackageValidationBaselineVersion>17.11.0-preview-24318-05</PackageValidationBaselineVersion>
```
Comment (f-alizada) [inline eng/Versions.props:6]: Should it be published?

Comment (AR-May) [reply eng/Versions.props:6]: We discussed which version to put here with @rainersigwald It is published internally, unfortunately, we cannot put here any version published publicly - the latest are 17.10 versions only and we need 17.11 here.

Comment (rainersigwald) [reply eng/Versions.props:6]: There's a follow-up item in the release checklist to bump this to the final released version when it's published to nuget.org.

**Review by f-alizada — APPROVED** [review]

**Comment by AR-May** [comment]
Deleted unnecessary compatibility suppressions according to the CI error.

**Pull request merged**